src/wally-connector.ts

```diff
@@ -183,6 +246,8 @@ class WallyConnector {
       if (resp && resp?.ok && resp?.status < 300) {
         const data = await resp.json();
         this.setAuthToken(data.token);
+        this.selectedAddress = data.wallet;
+        this.onTokenFetched && this.onTokenFetched(data.wallet);
```

Comment:
Q: Does the following work 🤔 ?

```suggestion
        this.onTokenFetched?(data.wallet);
```

Reply:
unfortunately not ☹️ 

![Screen Shot 2022-11-28 at 11 58 55 AM](https://user-images.githubusercontent.com/2947034/204369491-719051cb-720d-46b8-ab2d-e76d49f395e0.png)